### PR TITLE
Nunavut child benefit

### LINKED ISSUE #188: Nunavut child benefit
[Reference](https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/t4114/canada-child-benefit.html#toc20)

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #188

### CODE REVIEW

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base_component/base.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base_component/base.yaml:1]: Could we rename the folder/file `base/amount.yaml`

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working_component/single_child.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working_component/single_child.yaml:1]: Could this be a scale parameter `amount.yaml`?

Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working_component/single_child.yaml:1]: Rename folder `working`

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working_component/single_child.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Nunavut child benefit Territorial workers' max supplement for one child (per familty)
+values:
+  2022-07-01: 274.92 #22.91 monthly
+metadata:
+  unit: currency-CAD
+  label: Territorial workers' max supplement for one child
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working_component/single_child.yaml:6]: Similar for others
```suggestion
  label: Nunavut child benefit territorial workers' max supplement for one child
```

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working_component/single_child.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: Nunavut child benefit Territorial workers' max supplement for one child (per familty)
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working_component/single_child.yaml:1]: And elsewhere
```suggestion
description: Nunavut provides this maximum child benefit supplement for territorial workers with one child (per family).
```

File: policyengine_canada/tests/gov/provinces/nu/benefits/child_benefit/nu_child_benefit_base_component_base.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# - name: Nunavut one child
```
Comment (MaxGhenis) [inline policyengine_canada/tests/gov/provinces/nu/benefits/child_benefit/nu_child_benefit_base_component_base.yaml:1]: commented?

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base/amount.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+description: Nunavut provides this base component's base for each child in one family as part of the chile benefit.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base/amount.yaml:1]:
```suggestion
description: Nunavut provides this base component's base for each child in one family as part of the child benefit.
```

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base/amount.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+values:
+  2022-07-01: 330 #per child under age of 18 living at home
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base/amount.yaml:3]: Per yaml style guides like https://developers.home-assistant.io/docs/documenting/yaml-style-guide/#comments
```suggestion
  2022-07-01: 330 # Per child under age of 18 living at home
```

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base/phase_out_rate.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
+description: Nunavut child benefit's base component is phased out at this reduction rate.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base/phase_out_rate.yaml:1]:
```suggestion
description: Nunavut phases out its child benefit base component at this rate.
```

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working/single_child.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+description: Nunavut provides this maximum child benefit supplement for territorial workers with one child (per family).
+values:
+  2022-07-01: 274.92 #22.91 monthly
+metadata:
+  unit: currency-CAD
+  label:  Nunavut child benefit territorial workers' max supplement for one child
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working/single_child.yaml:6]:
```suggestion
  label: Nunavut child benefit territorial workers' max supplement for one child
```

File: policyengine_canada/tests/gov/provinces/nu/benefits/child_benefit/nu_child_benefit_base_component.yaml [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+- name: reduction over base
+  period: 2023
+  input:
+    nu_child_benefit_base_component_base: 500
+    nu_child_benefit_base_component_reduction: 1_500
+    province_code: NU
+  output:
+    nu_child_benefit_base_component: 0
+
```
Comment (MaxGhenis) [inline policyengine_canada/tests/gov/provinces/nu/benefits/child_benefit/nu_child_benefit_base_component.yaml:18]: remove spare lines

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working/single_child.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working/single_child.yaml:1]: Could we combine this and two_or_more_children into a scale parameter?

**Review by MaxGhenis — COMMENTED** [review]
nits

File: policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working/single_child.yaml
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/working/single_child.yaml:1]: Done!

**Pull request merged**